app/comparator.py

```
@@ -0,0 +1,134 @@
+            # Run the comparison script
+            result = subprocess.run(
+                [self.script_path, repo_path, old_commit, new_commit, repo_name],
+                capture_output=True,
+                text=True
+            )
+            
+            if result.returncode != 0:
+                raise Exception(f"Script failed: {result.stderr}")
+            
+            # Get the output file
+            current_date = datetime.now().strftime('%Y-%m-%d')
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)